ftplugin/latex-suite/folding.vim

```diff
@@ -257,6 +257,26 @@ function! MakeTexFolds(force, manual)
 	endif
 	" }}}
 
+	let g:Tex_FoldBeamerExtra=1
```

Comment:
With this line, one cannot disable this option. You should look how it is done for other options. Do we really need a new option? Maybe one could integrate this in Tex_FoldedMisc?

Reply:
I don't understand how the other options work to be honest. I thought you could change the value of the flag in one's .vimrc file.

Reply:
I need your help to do this. Maybe you can give me a few tips.

Reply:
I think it makes more since to have a separate flag, because these folding options are beamer-specific. I was thinking to have a flag you can turn on/off. Because if you are writing a non-beamer document, you might want to define the frame and column functions differently.

How do you turn off an item from Tex_FoldedMisc?

Reply:
You just have to remove it from the comma-separated list.